targets/TARGET_ARM_SSG/TARGET_CM3DS_MPS2/device/device_cfg.h

```diff
@@ -58,4 +58,8 @@
 #define ARM_UART3
 #define ARM_UART4
 
+/* SMSC9220 Ethernet */
+#define SMSC9220_ETH
+#define SMSC9220_Ethernet_Interrupt_Handler     ETHERNET_IRQHandler
```

Comment:
Why is this not using dynamic vectors (NVIC Set/Get vector) ?

Reply:
Yes, this will pull in a chunk of Ethernet driver code, even when the application isn't using networking. Even if ROM is not a concern for the CM3DS, it's a bad example for in-tree.